client-core/src/service/key_service.rs

```diff
@@ -20,6 +42,20 @@ where
         KeyService { storage }
     }
 
+    /// Generates keypair by wallet kinds recorded in sled storage
+    pub fn generate_keypair_auto(
```

Comment:
Can you rename this to `generate_keypair` and other one to `generate_keypair_basic`?

Reply:
generate_keypair works the same (normal),
generate_keypair_hd (for hd wallet),
generate_keypair_auto handles both of them

Reply:
i will change to generate_keypair -> generate_keypair_basic